gcp_airflow_foundations/source_class/generic_file_source.py

```diff
@@ -242,7 +242,7 @@ def load_to_landing(self, table_config, **kwargs):
             date_column = table_config.extra_options.get("sftp_table_config")["date_column"]
             gcs_bucket_prefix = file_source_config.gcs_bucket_prefix
             # bq load command if parquet
-            partition_prefix = ti.xcom_pull(key='partition_prefix', task_ids='ftp_taskgroup.load_sftp_to_gcs')
+            partition_prefix = ti.xcom_pull(key='partition_prefix', task_ids=f'{table_name}.ftp_taskgroup.load_sftp_to_gcs')
```

Comment:
for places where we cxom_pull or xcom_push, it may make sense to create a function for generating the task name
'load_sftp_to_gcs_task_name(table_name)'
and reuse it from both call places. Just to reduce change of typos.

Reply:
yep, makes sense